cachito/workers/pkg_managers/npm.py

```diff
@@ -276,10 +286,15 @@ def get_package_and_deps(package_json_path, package_lock_path):
         package_lock = json.load(f)
 
     package_lock_original = copy.deepcopy(package_lock)
-    name_to_deps, top_level_replacements = _get_deps(package_lock.get("dependencies", {}))
+    package = {"name": package_lock["name"], "type": "npm", "version": package_lock["version"]}
```

Comment:
We're using values from `package_lock` before `_get_deps` is called. I think that's ok because the only thing that may change withing `package_lock` is the `dependencies` which we do not use to construct the `package` dict.